Automattic/wp-calypso
# Display correct saved amount percentage when switching to annual plan

When you go through the process of buying a plan, the Plans page [monthly] shows you an annual purchase discount that is slight more than the actual discount. For example choosing the premium plan will provide a discount of 55.5% but it claims to be 56%:


Same applies to the other plans, so we will floor the discount rate rather than rounding it so it shows as:



Related to #65969

## Proposed Changes

* Display correct saving percentage by using floor rather than rounding.

## Testing Instructions

1. Before applying the patch, visit the plans page [http://calypso.localhost:3000/plans/monthly/siteName] while toggled to the monthly plans: we should see ![szNeK2.png](https://user-images.githubusercontent.com/552016/230504833-bbdb9800-8c95-4600-bd9c-2bc4e403602f.png)
2. Note that the amount saved by the customer is informed for premium plan as  `Save 56% by paying annually`, similarly it is also mentioned the saving amounts for other plans.
3. Apply the patch.
4. Notice a change/reduction in discount percentage to this  ![Lz6sYZ.png](https://user-images.githubusercontent.com/552016/230504893-de93625b-d02a-4e1c-a608-c262cce6e042.png)
